app/scripts/modules/core/src/pipeline/service/execution.service.ts

```diff
@@ -33,11 +33,11 @@ export class ExecutionService {
       return this.getFilteredExecutions(applicationName, this.activeStatuses, this.runningLimit);
     }
 
-    private getFilteredExecutions(applicationName: string, statuses: string[], limit: number, pipelineConfigIds: string[] = null): IPromise<IExecution[]> {
+    private getFilteredExecutions(applicationName: string, statuses: string[], limit: number, pipelineConfigIds: string[] = null, expand = false): IPromise<IExecution[]> {
```

Comment:
Is there a reason the new parameter doesn't have an explicit type?

Reply:
typescript infers the type when it's trivial to do so from the default value (and our linter will complain if the annotation is added needlessly).

Reply:
👍